zipline/utils/calendars/trading_calendar.py

```diff
@@ -684,17 +685,22 @@ def minute_to_session_label(self, dt, direction="next"):
             The label of the containing session.
         """
 
-        idx = searchsorted(self.market_closes_nanos, dt.value)
+        if type(dt) is pd.Timestamp:
```

Comment:
Switching to something else would be fine. Do you recommend `isinstance`? I don't know how the `@preprocess` stuff works, so would that take care of it as well? Would it be preferable to do it that way?